test/unit/client_test.coffee

```diff
@@ -107,6 +108,12 @@ describe "Client", ->
         error = _err
       return error
 
+    it "returns promise and resolves it", ->
+      promise = client.push(exception)
```

Comment:
It would be enough, but callbacks are considered a bad practise nowadays and `Promise` API is going to be a part of ES6 spec.